discretize/mixins/vtkModule.py

```diff
@@ -355,44 +355,57 @@ def toVTK(mesh, models=None):
         """
         # TODO: mesh.validate()
         converters = {
-            'tree' : vtkInterface.__treeMeshToVTK,
-            'tensor' : vtkInterface.__tensorMeshToVTK,
-            'curv' : vtkInterface.__curvilinearMeshToVTK,
-            #TODO: 'CylMesh' : vtkInterface.__cylMeshToVTK,
+            'tree' : InterfaceVTK.__tree_mesh_to_vtk,
+            'tensor' : InterfaceVTK.__tensor_mesh_to_vtk,
+            'curv' : InterfaceVTK.__curvilinear_mesh_to_vtk,
+            #TODO: 'CylMesh' : InterfaceVTK.__cyl_mesh_to_vtk,
             }
         key = mesh._meshType.lower()
         try:
             convert = converters[key]
         except KeyError:
             raise RuntimeError('Mesh type `{}` is not currently supported for VTK conversion.'.format(key))
-        # Convert the data object then attempt a wrapping with `vtki`
+        # Convert the data object then attempt a wrapping with `pyvista`
         cvtd = convert(mesh, models=models)
         try:
-            import vtki
-            cvtd = vtki.wrap(cvtd)
+            import pyvista
+            cvtd = pyvista.wrap(cvtd)
         except ImportError:
-            warnings.warn('For easier use of VTK objects, you should install `vtki` (the VTK interface): pip install vtki')
+            warnings.warn('For easier use of VTK objects, you should install `pyvista` (the VTK interface): pip install pyvista')
         return cvtd
 
+    def to_vtk(mesh, models=None):
+        """Convert this mesh object to it's proper VTK or ``pyvista`` data
+        object with the given model dictionary as the cell data of that dataset.
+
+        Parameters
+        ----------
+
+        models : dict(numpy.ndarray)
+            Name('s) and array('s). Match number of cells
+
+        """
+        return InterfaceVTK.toVTK(mesh, models=models)
```

Comment:
Do we want to encourage users to use the `to_vtk` rather than `toVTK`? if so do you think we should move the code here and perhaps throw a warning for those using `toVTK`?

Reply:
If the `discretize` API is heading towards pep8, I can add a warning to these methods

Reply:
implemented in b4370a7